histomicstk/features/ComputeHaralickFeatures.py

```diff
@@ -62,89 +57,149 @@ def ComputeHaralickFeatures(im_label, im_intensity, offsets=None, num_levels=8,
+
+        .. math::
+
+            DifferenceVariance = {\rm variance \ of ~} p_{x-y}, \qquad where \\
+
+            p_{x-y}(k) =
+            \sum_{i,j=0}^{levels-1} \delta_{|i-j|, k} p(i,j)
 
     Haralick.DifferenceEntropy.Mean : float
-        Mean of differnce entropy of object pixels.
+        Mean and range of the difference entropy feature for GLCMS of all
+        offsets. It is computed as follows:
 
-    Haralick.IMC1.Mean : float
-        Mean of the first information measures of correlation of object
-        pixels.
+        .. math::
 
-    Haralick.IMC2.Mean : float
-        Mean of the second information measures of correlation of object
-        pixels.
+            DifferenceEntropy = {\rm entropy \ of ~} p_{x-y}
 
-    Haralick.ASM.Range : float
-        Range of angular second moment of object pixels.
+    Haralick.IMC1.Mean, Haralick.IMC1.Range : float
+        Mean and range of the first information measure of correlation
+        feature for GLCMs of all offsets. It is computed as follows:
 
-    Haralick.Contrast.Range : float
-        Range of contrast of object pixels.
+        .. math::
 
-    Haralick.Correlation.Range : float
-        Range of correlation of object pixels.
+            IMC1 = \frac{HXY - HXY1}{\max(HX,HY)}, \qquad where \\
 
-    Haralick.SumofSquar.Range : float
-        Range of sum of squares of object pixels.
+            HXY = -\sum_{i,j=0}^{levels-1} p(i,j) \log(p(i,j)) \\
 
-    Haralick.IDM.Range : float
-        Range of inverse difference moment of object pixels.
+            HXY1 = -\sum_{i,j=0}^{levels-1} p(i,j) \log(p_x(i) p_y(j)) \\
 
-    Haralick.SumAverage.Range : float
-        Range of sum avarage of object pixels.
+            HX = -\sum_{i=0}^{levels-1} p_x(i) \log(p_x(i))    \\
 
-    Haralick.SumVariance.Range : float
-        Range of sum variance of object pixels.
+            HY = -\sum_{j=0}^{levels-1} p_y(j) \log(p_y(j))    \\
 
-    Haralick.SumEntropy.Range : float
-        Range of sum entropy of object pixels.
+            p_x(i) = \sum_{j=1}^{levels} p(i,j) \\
 
-    Haralick.Entropy.Range : float
-        Range of entropy of object pixels.
+            p_y(j) = \sum_{j=1}^{levels} p(i,j)
 
-    Haralick.Variance.Range : float
-        Range of variance of object pixels.
+    Haralick.IMC2.Mean : float
+        Mean and range of the second information measure of correlation
+        feature for GLCMs of all offsets. It is computed as follows:
 
-    Haralick.DifferenceEntropy.Range : float
-        Range of differnece entropy of object pixels.
+        .. math::
 
-    Haralick.IMC1.Range : float
-        Range the first information measures of correlation of object
-        pixels.
+            IMC2 = [1 - \exp(-2(HXY2 - HXY))]^{1/2}, \qquad where \\
 
-    Haralick.IMC2.Range : float
-        Range the second information measures of correlation of object
-        pixels.
+            HXY2 = -\sum_{i,j=0}^{levels-1} p_x(i) p_y(j) \log(p_x(i) p_y(j))
```

Comment:
Added clearer documentation for all features. You can put latex equations in the docstring like this.